### PR TITLE
Add a Russian translation

### LINKED ISSUE #86: Transalte  markdowntutorial.com to Russian
I want to contribute a Russian translation

### PR DESCRIPTION
**Pull request by hexagonrecursion**
Fixes #86 

Note: this PR includes #87

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,7 +3,7 @@
 /bin
 /vendor/ruby
 /vendor/gems/
-/vendor/cache/ruby/
+/vendor/cache/
 
 .DS_Store
```
Comment (Eric5545) [inline .gitignore:8]: ``

File: .gitignore [hunk 2]
```diff
@@ -3,7 +3,7 @@
 /bin
```
Comment (Eric5545) [inline .gitignore:3]:
```suggestion
> #-  /bin_```

```_
```

**Comment by gjtorikian** [comment]
Is this PR ready for review?

**Comment by hexagonrecursion** [comment]
No.

**Comment by hexagonrecursion** [comment]
I'm sorry, but I'm unlikely to get back to finishing this any time soon

**Review by Shahidul783 — APPROVED** [review]

**Review by Shahidul783 — APPROVED** [review]

**Pull request closed without merging**